composer.json

```diff
@@ -12,13 +12,13 @@
         { "name": "Jeremy Mikola", "email": "jmikola@gmail.com" }
     ],
     "require": {
-        "php": ">=5.5.0",
-        "symfony/framework-bundle": "~2.7|~3.0",
-        "symfony/console": "~2.7|~3.0",
-        "symfony/form": "~2.7|~3.0",
-        "symfony/property-access": "~2.7|~3.0",
+        "php": ">=5.6.0",
+        "symfony/framework-bundle": "~2.7|~3.1",
```

Comment:
Can you use `||` to separate different version numbers?
https://getcomposer.org/doc/articles/versions.md#range